src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -60,7 +60,61 @@ def test_WINNF_FT_S_SCS_4(self):
   def test_WINNF_FT_S_SCS_5(self):
     """New registration with TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 cipher.
 
-    Checks that SAS UUT response satisfy specific security conditions.
+    Checks that SAS UUT response satisfy cipher security conditions.
     Checks that a CBSD registration with this configuration succeed.
     """
     self.doTestCipher('ECDHE-RSA-AES128-GCM-SHA256')
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_6(self):
+    """Unrecognized root of trust certificate presented during registration.
+    Checks that SAS UUT responds with fatal alert with unknown_ca.
+    """
+    device_cert = os.path.join('certs', 'unrecognized_device.cert')
```

Comment:
Suggestion: as we start using a lot of certificates, perhaps introduce a method in security_testcase.py, in class SecurityTestCase:

```
def getCertFilename(self, cert_name)
"""Returns the absolute path of the file corresponding to the given |cert_name|.
"""
    harness_dir = os.path.dirname(
              os.path.abspath(inspect.getfile(inspect.currentframe())))
    return os.path.join(harness_dir, 'certs', cert_name)
```
And then use it here and below as:
```
    device_cert = self.GetCertFilename('unrecognized_device.cert')
```
The advantage is that we abstract the real location of the certificates: it become easier to mock/overload them to run the testcases against our real SAS implementation.